mailbox/jpa/pom.xml

```diff
@@ -31,6 +31,10 @@
     <packaging>jar</packaging>
     <name>Apache James :: Mailbox :: JPA</name>
 
+    <properties>
```

Comment:
How does this behave in IDEs ? does it mean we need to have both jdk 11 and jdk21 installed to be able to build james ?

Reply:
> does it mean we need to have both jdk 11 and jdk21 

I believe we only need jdk21 installed.
I set IDE and terminal jdk to jdk21 only. Then IDE build and terminal build are both fine.